packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -0,0 +1,319 @@
+import '@agoric/zoe/exported.js';
+import '@agoric/zoe/src/contracts/exported.js';
+import '@agoric/governance/exported.js';
+
+import { M, makeScalarBigMapStore, provide } from '@agoric/vat-data';
+import { AmountMath, AmountShape } from '@agoric/ertp';
+import { Far } from '@endo/marshal';
+import { mustMatch } from '@agoric/store';
+import { observeNotifier } from '@agoric/notifier';
+
+import {
+  atomicRearrange,
+  ceilMultiplyBy,
+  floorDivideBy,
+  makeRatioFromAmounts,
+  multiplyRatios,
+  ratioGTE,
+} from '@agoric/zoe/src/contractSupport/index.js';
+import { TimeMath } from '@agoric/time';
+import { E } from '@endo/captp';
+import { makeTracer } from '@agoric/internal';
+
+import { makeDiscountBook, makePriceBook } from './discountBook.js';
+import {
+  AuctionState,
+  isDiscountedPriceHigher,
+  makeRatioPattern,
+} from './util.js';
+import { keyToTime } from './sortedOffers.js';
+
+const { Fail } = assert;
+
+/**
+ * @file The book represents the collateral-specific state of an ongoing
+ * auction. It holds the book, the lockedPrice, and the collateralSeat that has
+ * the allocation of assets for sale.
+ *
+ * The book contains orders for a particular collateral. It holds two kinds of
+ * orders: one has a price in terms of a Currency amount, the other is priced as
+ * a discount (or markup) from the most recent oracle price.
```

Comment:
please enumerate and name,
```suggestion
 * orders:
 *   - Absolute: priced in terms of a Currency amount
 *   - Coefficient: a discount (or markup) from the most recent oracle price
```

Reply:
in this file I use `discountBook` and `priceBook`.